src/sidebar/components/AnnotationView.tsx

```diff
@@ -57,7 +57,7 @@ function AnnotationView({ loadAnnotationsService, onLogin }) {
         // Make the full thread of annotations visible. By default replies are
         // not shown until the user expands the thread.
         annots.forEach(annot =>
-          store.setExpanded(/** @type {string} */ (annot.id), true)
+          annot.id && store.setExpanded(annot.id, true)
```

Comment:
I'm not sure what's the best way to do this. Previously we seemed to be neglecting the fact that `annot.id` can be undefined by doing an explicit casting.

Other possible approaches are:

```ts
if (annot.id) {
  store.setExpanded(annot.id, true)
}
```

```ts
store.setExpanded(annot.id ?? '', true)
```

```ts
store.setExpanded(annot.id!, true)
```